index.js

```diff
@@ -11,7 +11,7 @@ module.exports = function (str) {
   // U+2026  0x85  … â€¦   %E2 %80 %A6
   .replace(/â€¦/g, '…')
   // U+2020  0x86  † â€  %E2 %80 %A0
-  .replace(/â€/g, '†')
+  .replace(/â€\u00A0/g, '†')
```

Comment:
I checked and the characters given back for this symbol was actually one more. This is important as otherwise this regexp is blocking all other regexps further down that also contain `â€`.